lib/vsc/administration/vo.py

```diff
@@ -159,14 +158,15 @@ def sharing_group(self):
                                                 VO_SHARED_PREFIX_BY_INSTITUTE[self.vo.institute['name']])
             self._sharing_group_cache = mkVscAutogroup(
                 whenHTTPErrorRaise(self.rest_client.autogroup[group_name].get,
-                                   "Could not get autogroup %s details" % group_name)[1])
+                                   f"Could not get autogroup {group_name} details")[1])
 
         return self._sharing_group_cache
 
     @property
     def data_sharing(self):
         return self.vo_data_shared_quota is not None
 
+    @property
```

Comment:
@itkovian I don't know what happened here, but tests were failing on both py36 and 39. It seems this line was forgotten? But I don't understand how tests could have worked before without it.

(also a bunch of attributes were missing in test data)

Reply:
ah, I understand how it got missed. it only fails when you have a object with missing attributes (isactive was missing in some testdata, triggering the bug).

Reply:
it either the `@property` or the `error_vos[vo.vo_id] = vo.members` that needs to become a function call (`.members()`) on line 568

Reply:
@stdweird  in `process_vos` on line 568, it has `error_vos[vo.vo_id] = vo.members`. it should be `error_vos[vo.vo_id] = vo.members()`.

Reply:
hmm, the more I think about it, the I feel like the 2nd option is the better one.

Reply:
I switched it around and fixed the vo.members call